src/backend/mock.mk

```diff
@@ -38,9 +38,6 @@ EXCL_OBJS=\
 # of the test programs. Feel free to link them back (i.e. remove them from
 # this exclusion list) as needed.
 EXCL_OBJS+=\
-	src/backend/access/hash/hash.o \
-	src/backend/access/hash/hashsearch.o \
-	\
```

Comment:
Why we remove these

Reply:
Because hash index access method has changed some, so for run test succssfully, it has to remove from exclude test files.